app/models/organization.rb

```diff
@@ -57,12 +64,19 @@ def primary_address
     addresses.first
   end
 
-  def open_orders
-    @_open_orders ||= orders.select(&:open?)
-  end
-
   private
 
+  def ensure_no_open_orders
+    return unless open_orders.present?
+
+    msg = <<-eos
+      '#{name}' was unable to be deleted. We found the following open orders:
+      #{open_orders.map(&:id).to_sentence}
+    eos
+
+    raise DeletionError, msg
```

Comment:
Is this referencing the class correctly? It looked like you wrapped it in a StockAidException namespace...? However, I think dropping the namespace would probably be a little easier anyways.